packages/fluentui/react-bindings/src/hooks/useFrameFocusDispatch.ts

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+
+export const FUI_FRAME_EVENT = 'fuiframefocus';
+
+export const useFrameFocusDispatch = (enableFrameFocusDispatch: boolean, $document: Document) => {
+  const timeoutRef = React.useRef<ReturnType<typeof setTimeout>>();
+
+  React.useEffect(() => {
+    if (enableFrameFocusDispatch) {
+      timeoutRef.current = setInterval(() => {
+        if ($document.activeElement?.tagName === 'IFRAME') {
```

Comment:
I would save this into a const before using it to avoid the active element changing due to race condition.

```suggestion
        const activeElement = document.activeElement;
        if ($document.activeElement?.tagName === 'IFRAME') {
```